### ISSUE TITLE
Force PNG fallback for specific pages

### ISSUE THREAD

**Issue by lakenen** [box/viewer.js#166]
See: https://github.com/box/viewer.js/issues/147#issuecomment-66936772 for more info.

**Comment by chipkellam** [comment]
@lakenen With the latest library (didn't try earlier versions), I am getting 404 for all pages after page 1 when using the PNG fallback... any ideas?

**Comment by lakenen** [comment]
Yes! You will need to specify the non_svg flag for conversions, otherwise only page one will have a PNG generated (used for thumbnails). https://developers.box.com/view/#non-svg
